data/lemonSaves.txt

```diff
@@ -0,0 +1,2 @@
+D|false| help me |2023-03-22
+E|false| murder me |2023-04-03|2023-04-03
```

Comment:
Aww, those task names :c

---

Generally it's not a good idea to not push data files like this to a remote repository. Otherwise if you had actual tasks saved to the file, strangers like me can see your *potentially private* data :O

You can consider adding the following to the `.gitignore` file in your root directory:

```bash
# current content of .gitignore
# ...
/data/
```

So that when you add files to a git commit, git knows to ignore the directory `./data/` :D